frontend/src/components/intake/ShasIntakeForm.vue

```diff
@@ -89,15 +103,50 @@ function displayErrors(a: any) {
 }
 
 function confirmSubmit(data: any) {
+  const tempData = Object.assign({}, data);
+  delete tempData['addressSearch'];
```

Comment:
Just a mental note, not something to be done, its tech-debty - but we should be creating form submission types to avoid using `any` on submits.